lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -701,4 +704,4 @@ extern "C" {
 }
 
 /// Payload is a generic type used to represent raw incoming RDP messages for parsing.
-pub(crate) type Payload = Cursor<Vec<u8>>;
+pub(crate) type RawPayload = Cursor<Vec<u8>>;
```

Comment:
I don't love the name `RawPayload`. To me, `Payload` already means an arbitrary stream of bytes. So a raw payload, is what? Raw arbitrary bytes?

Reply:
Yes this change isn't needed. In a previous iteration, I had a new struct that I thought was more appropriately named `Payload` and so I'd changed it. Reverting.